### ISSUE TITLE
VOL column in DCA (and maybe other) log off by around 4000?

### ISSUE THREAD

**Issue by Legedric** [taniman/profit-trailer#895]
Hey guys,

today I looked at my DCA log and saw that my VOL number for BTC-XLM on Bittrex seemed way off the chart:
![image](https://user-images.githubusercontent.com/8308141/35511907-16cfcba2-04fe-11e8-9f6f-1cec21f24489.png)

I always thought it would be the 24h volume but looking on Bittrex I saw that this cannot be the case:
![image](https://user-images.githubusercontent.com/8308141/35511940-3c2f845a-04fe-11e8-875c-62b0345ae0d6.png)

So I went to the wiki to make sure I understand it correctly and there I found this:
![image](https://user-images.githubusercontent.com/8308141/35511976-5a3ce776-04fe-11e8-97e3-e6d276d4f29d.png)

Well, to be honest. I think the wiki is not correct here as I do not believe that the bot is considering buying 5299 units of XLM in DCA when I only own around 340 units until now. Using double down it would consider buying 680 maybe but most definitley not 5k or more.

So what exactly does the VOL column represent? I somehow believe it does show the 24 BTC volume for each line as most entries are somewhere between 200 and 600 right now on Bittrex.
But for this particular market (BTC-XLM) it seems to be off by around 4000 in my monitoring DCA log.

**Comment by taniman** [comment]
You are correct. This is a wrong description of VOL. Can you provide the link to fix this this?

VOL is the current 24h BTC Volume of the coin

**Comment by Legedric** [comment]
The wrong wiki entry is found here:
https://wiki.profittrailer.io/doku.php/web_interface_guide#main_area_-_dca_log

@taniman But if it is the 24h BTC volume for that coin, how is it possible that it did not change at all for almost a whole day now and it does not fit to the volume shown on the exchange itself (Bittrex in that case) even remotely close?

This is a recent screenshot from my PT monitor:
![grafik](https://user-images.githubusercontent.com/8308141/35548268-9fc2dc12-057e-11e8-95f3-1657cd85b5e3.png)

This from Bittrex directly:
![grafik](https://user-images.githubusercontent.com/8308141/35548286-b92b85d2-057e-11e8-8bdf-4e3154cafb6d.png)

**Comment by Legedric** [comment]
I just restarted my bot and now it seems to have updated the VOL column.

So somehow this seems to have frozen although the bot was still running and giving heartbeats.

**Comment by GGaulin** [comment]
@taniman The problem of the GUI not updating and requiring a restart to clear the issue seems to be happening more frequently. I'm planning on setting up WireShark to record bot to Binance API communications during this time. I will send you the log when available. Let me know if there is anything else you would like to help identify root cause.

**Comment by taniman** [comment]
@Legedric what version of the bot?
@GGaulin version .14 should be okay

**Comment by Legedric** [comment]
@taniman It happened with .12 at first but .14 is showing the same "frozen" volume in my DCA log. It somehow freezes the volume column at some point (for Bittrex at least).

**Comment by Dehumanizer77** [comment]
I am experiencing the same problem, volumes are always frozen until restart, then it's frozen again until next restart. .14 and bittrex too
